release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1153,6 +1153,49 @@ The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for ex
 ==== Hyperthreading-aware CPU manager policy (Technology Preview)
 Hyperthreading-aware CPU manager policy in {product-title} is now available without the need for extra tuning. The cluster administrator can enable this feature if required. Hyperthreads are abstracted by the hardware as logical processors. Hyperthreading allows a single physical processor to execute two heavyweight threads (processes) at the same time, dynamically sharing processor resources.
 
+[id="ocp-4-10-topology-aware-lifecycle-operator"]
+==== Updating managed clusters with Topology Aware Lifecycle Operator (Technology Preview)
+
+You can now use the Topology Aware Lifecycle Operator to perform updates on multiple single node Openshift (SNO) clusters by using {rh-rhacm-first} policies. From {product-title} {product-version}, TALO is available in the OperatorHub. For more information, see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.html#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator configuration].
```

Comment:
Consider including the the acronym form in the first instance of Topology Aware Lifecycle Operator so that it is put in context at its later use. 

Further consider removing SNO acronym. ([in progress PR for glossary entry](https://github.com/openshift/openshift-docs/pull/41814/files#diff-4e988659a0521aa80596c74e27373e09aaf20218c945862743f6c72288952045R645))
```suggestion
You can now use the Topology Aware Lifecycle Operator (TALO) to perform updates on multiple single node Openshift clusters by using {rh-rhacm-first} policies. From {product-title} {product-version}, TALO is available in the OperatorHub. For more information, see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.html#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator configuration].
```